test.sh

```diff
@@ -14,8 +14,11 @@ end flake8
 
 start python
 pushd hubmap
-python manage.py test
+python3 manage.py test
 # TODO: Django tests
+service postgresql start
+python3 manage.py makemigrations
```

Comment:
It definitely isn't appropriate to run `makemigrations` here; migrations must be committed with the code that requires them.